package.json

```diff
@@ -100,10 +100,10 @@
   "engines": {
     "npm": "please-use-pnpm",
     "node": ">=18",
-    "pnpm": "8.15.6"
+    "pnpm": "8.15.7"
```

Comment:
I get we want to use https://pnpm.io/installation#using-corepack but do we need to have this here?

```suggestion
```

Maybe with pnpm v9 we don't need this anymore: https://github.com/pnpm/pnpm/releases/tag/v9.0.0-rc.2

> pnpm will now check the package.json file for a packageManager field. If this field is present and specifies a different package manager or a different version of pnpm than the one you're currently using, pnpm will not proceed. This ensures that you're always using the correct package manager and version that the project requires.

Reply:
We enforce a specific version with the engines field. In case someone tries to install who is not using corepack, or an older (pre-corepack) pnpm version.

Reply:
Ok, this makes sense.

I would wait to see that the next upgrades of pnpm go smoothly, and once they do go all-in to propagate this in all repositories.

Reply:
Looks like it does: https://github.com/mui/mui-toolpad/pull/3499